inst/qml/common/GlmResidualAnalysisPlotsComponent.qml

```diff
@@ -31,32 +31,32 @@ Group
 	Group
 	{
 		title: qsTr("Deviance Residuals")
-		CheckBox { name: "devResVsYPlot";	label: qsTr("Residuals vs. fitted")		}
-		CheckBox { name: "devResVsXPlot";	label: qsTr("Residuals vs. predictor")	}
-		CheckBox { name: "devResQqPlot";	label: qsTr("Q-Q plot")					}
+		CheckBox { name: "devianceResidualVsFittedYPlot";	label: qsTr("Residuals vs. fitted")		}
```

Comment:
what's the reason of using `devianceResidualVsFittedYPlot` and not `devianceResidualVsFittedYPlot`?

Reply:
What's the difference between the two?

Reply:
nothing, because I made a typo, sorry. I meant `devianceResidualVsFittedPlot` (dropping the `Y`)

Reply:
thx! will do